README.md

```diff
@@ -1,25 +1,21 @@
 ![Image of Gopher flying](https://github.com/glycerine/zygomys/blob/master/media/high_altitude_gopher.png)
 
-# zygomys - embedded scripting language for Go.
+# zygomys - An embedded scripting language for Go.
 
-### quickly create a mini-language to configure your project
+### Why use zygomys?
 
-zygomys is an embeddable scripting language. It is a modernized Lisp with an object-oriented flavor, and
-provides an interpreter and REPL (Read-Eval-Print-Loop;
-that is, it comes with a command line interactive interface).
-
-### where did the name zygomys come from?
-
-zygomys is a contraction of Zygogeomys, [a genus of pocket gophers. The Michoacan pocket gopher (Zygogeomys trichopus) finds its natural habitat in high-altitude forests.](https://en.wikipedia.org/wiki/Michoacan_pocket_gopher)
-
-The term is also descriptive. The stem `zygo` comes from the Greek for yoke, indicating a pair or a union of two things, and `mys` comes from the Greek for mouse. The union of Go and Lisp in a small cute package, that is zygomys.
+zygomys is an embeddable scripting language that can be used to 
+quickly configure your Go project.  
+It allows you to create a Domain Specific Language to drive your Go programs 
+with minimal fuss and maximum convenience.
 
-## tell me more
+### What is zygomys?
 
-zygomys allows you to create a Domain Specific Language to drive
-your program with minimal fuss and maximum convenience.
+zygomys is a modernized Lisp with an object-oriented flavor, and
+provides an interpreter and REPL (Read-Eval-Print-Loop;
+that is, it comes with a command line interactive interface).
 
-It is written in Go and plays easily with Go programs
+It is written in Go and plays nicely with Go programs
```

Comment:
I swapped out "easily" for "nicely" riffing off of Rich Hickey's talk "Simple Made Easy" - zygomys inter-operates with Go programs well and in a simple way, but using zygomys may not be easy for someone who's first introduction to Lisp is zygomys.